core/schema/src/main/liquibase/28.0.0/changelog.xml

```diff
@@ -559,7 +559,7 @@
           WHERE e.extname LIKE 'timescaledb';
         </sqlCheck>
         <sqlCheck expectedResult="0">
-          SELECT COUNT(table_name) FROM timescaledb_information.hypertable WHERE table_name = 'bmp_ip_rib_log';
```

Comment:
view and column names are different in timescaledb versions, this upgrades naming to timescaledb 2.x